### ISSUE TITLE
write a first draft of the QEP to remove GeoNode from core QGIS

### ISSUE THREAD

**Issue by ricardogsilva** [kartoza/qgis_geonode#154]

**Comment by ricardogsilva** [comment]
@Samweli 

I've started this up with PR #165 
Not sure if you already have any text for the QEP or not. If you do, feel free to edit the PR. Otherwise I will continue to work on it until it feels ready to ask for a review from @timlinux.

**Comment by Samweli** [comment]
@ricardogsilva yes I did started working on this, see https://github.com/Samweli/qgis_geonode/tree/qep. You can take anything from there to add to the PR https://github.com/kartoza/qgis_geonode/pull/165.
